add task solution

Khmeletskyy opened the pull request.

Khmeletskyy committed:
add task solution

Viktor-Kost commented on src/index.html:
```diff
@@ -7,8 +7,48 @@
+          <li class="nav__item">
+            <a class="nav__link" href="#video">Video</a>
+          </li>
+
+        </ul>
+      </nav>
+    </header>
+
+
```
```suggestion
        </ul>
      </nav>
    </header>
```

Viktor-Kost commented on src/style.css:
```diff
@@ -1,3 +1,72 @@
+  justify-content: space-between;
+}
+
+.nav__item {
+  margin: 0 0 0 20px;
+}
```
```suggestion
```

Viktor-Kost commented on src/style.css:
```diff
@@ -1,3 +1,72 @@
+  color: var(--link-main-color);
+}
+
+.nav__item:first-child {
+  margin-left: 0;
+}
```
```suggestion
.nav__item:not(:first-child) {
  margin-left: 20px;
}
```

Viktor-Kost commented on src/style.css:
```diff
@@ -1,3 +1,72 @@
+.header__logo {
+  width: 40px;
+  height: 40px;
+  display: flex;
+  justify-content: center;
```
you can just make as `block`

Viktor-Kost reviewed (changes requested):
Great job!

Something went wrong. Since you DEMO looks like
<img width="747" alt="Screenshot 2023-12-09 at 22 59 38" src="https://github.com/mate-academy/layout_moyo-header/assets/117587463/f232e8dc-0e3a-4b0e-88ad-5f4bd2672080">

Feel free to ask at fe_chat we will help you

Khmeletskyy committed:
- [DEMO LINK](https://Khmeletskyy.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://Khmeletskyy.github.io/layout_moyo-header/report/html_report/)
